src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/TypeComponentProps.ts

```diff
@@ -5,5 +5,5 @@ export interface TypeComponentProps {
   onChange?: (nextValue: any) => void;
   structureDefinition: any;
   fieldRequired: boolean;
-  signed?: boolean;
+  unsignedInt?: boolean;
```

Comment:
I don't think this is the right place for this prop, as TypeComponentProps is meant to be common to all of the type components and `unsignedInt` isn't a common prop. You can extend the TypeComponentProps interface for more specific component props
![image](https://github.com/user-attachments/assets/31486387-9af2-45c5-b27d-674430b846db)

Reply:
@nmorasb Ok thanks Nate, will do.

Reply:
@nmorasb Just pushed in the changes